nethttp/server.go

```diff
@@ -52,6 +54,15 @@ func MWSpanObserver(f func(span opentracing.Span, r *http.Request)) MWOption {
 	}
 }
 
+// MWURLTag returns a MWOption that uses given function f
+// to set the span's http.url tag. Can be used to change the default
+// http.url tag, eg to redact sensitive information.
+func MWURLTag(f func(u *url.URL) string) MWOption {
```

Comment:
please change to MWURLTagFunc, to match OperationNameFunc

Reply:
Done, thanks.